python3/src/stlib3/physics/rigid/RigidObject.py

```diff
@@ -48,19 +49,22 @@ def RigidObject(name="RigidObject",
 
     #### mechanics
     object = Sofa.Core.Node(name)
+
     if(parent != None):
         parent.addChild(object)
 
-    plugins = ['SofaRigid']
+    #plugins = ['SofaRigid']
     object.addObject('MechanicalObject',
                       name="mstate", template="Rigid3",
                       translation2=translation, rotation2=rotation, showObjectScale=uniformScale)
 
     object.addObject('UniformMass', name="mass", vertexMass=[totalMass, volume, inertiaMatrix[:]])
 
-    if not isAStaticObject:
-        plugins.append('SofaConstraint')
-        object.addObject('UncoupledConstraintCorrection')
+
+    #if not isAStaticObject:
+        #plugins.append('SofaConstraint')
+        #object.addObject('EulerImplicitSolver')
+        #object.addObject('CGLinearSolver')
```

Comment:
I'm not sure why this has been removed? @CorentinLanguepin @Younesssss 
same for the `RequiredPlugin`.